src/Microsoft.Extensions.CodeGeneration.EntityFrameworkCore/EntityFrameworkServices.cs

```diff
@@ -245,12 +243,11 @@ public class EntityFrameworkServices : IEntityFrameworkService
             out Type dbContextType, 
             out Type modelType)
         {
-            //TODO: @prbhosal Figure out how to lookup the correct project here.             
             var projectCompilation = _workspace.CurrentSolution.Projects
-                //.Where(project => project.Name == _environment.ApplicationName)
+                .Where(project => project.AssemblyName == _environment.ApplicationName)
                 .FirstOrDefault()
```

Comment:
`FirstOrDefault` takes a predicate, no need to do the `where` above. Also, is "project" a struct? If it can't find one I'd imagine it exploding with null ref.